charts/cassandra-migrations/templates/migrate-schema.yaml

```diff
@@ -7,15 +7,10 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
-{{- if .Values.hook.enabled }}
   annotations:
-    # when hook.enabled=true (default), this chart does not work standalone, but is intended as
-    # either a 'post'-subchart to a cassandra chart or a 'pre'-subchart to wire-server
-    # For the meaning of hooks, see https://docs.helm.sh/developing_charts/#hooks
-    "helm.sh/hook": {{ .Values.hook.type }}-install,{{ .Values.hook.type }}-upgrade
+    "helm.sh/hook": pre-install,pre-upgrade
```

Comment:
This is the way we've used it thus far, it's fine to hardcode it now; variables seem to have been premature optimization.